cmd/internal/ecs/add.go

```diff
@@ -943,11 +944,38 @@ func makeAgentContainerDefinition(
 	addOptToEnv("POSTMAN_ECS_SERVICE", ecsService)
 	addOptToEnv("POSTMAN_ECS_TASK", ecsTaskDefinitionFamily)
 
+	// Pass apidump flags as it is, the apidump command will parse them.
+	// We are already handling the default values in apidump command
 	entryPoint := []string{
 		"/postman-insights-agent",
 		"apidump",
 		"--project",
 		projectId,
+		"--filter",
+		filterFlag,
+		"--rate-limit",
+		strconv.FormatFloat(rateLimitFlag, 'f', -1, 64),
+		"--stats-log-delay",
+		strconv.Itoa(statsLogDelay),
```

Comment:
I think we should avoid adding new "always present" parameters like this.  Here's my reasoning:

1) It clutters up the command line, and we already have people asking us to take the command line via environment variables or config file.  Shouldn't give the impression that there is a lot of information here.

2) --rate-limit in particular is something where we may want to switch to a much different approach (a global rate limit shared among agents) and having it present for every agent install makes that transition harder.

3) Changes to the default behavior should be applied to old apidump installation when they upgrade to a new agent version.